text-document.rkt

```diff
@@ -73,12 +75,24 @@
 (define (start/end->range doc start end)
   (Range #:start (abs-pos->pos doc start) #:end (abs-pos->pos doc end)))
 
+(define-json-expander ConfigurationItem
+  [scopeUri string?]
+  [section string?])
+
+(define (fetch-configuration uri)
+  (send-request 0 "workspace/configuration"
+    (hasheq 'items (list (ConfigurationItem #:scopeUri uri #:section "racket-langserver")))
+    (λ (configuration)
+      ; TODO: apply configuration
```

Comment:
We can split the logic from function `didChangeConfiguration` in `workspace.rkt`, and reuse it here.